### ISSUE TITLE
Model weights

### ISSUE THREAD

**Issue by mrgiba** [tyomj/product_detection#7]
Hello,

Are the model weights available somewhere ?

Thank you

**Comment by tyomj** [comment]
Hi,

Yes, they are, please take a look at the comment [here](https://github.com/tyomj/product_detection/issues/2#issuecomment-774687585).

**Comment by mrgiba** [comment]
Awesome

And is there any caveat for doing transfer learning using that pre trained model as a base ?

Thanks

**Comment by tyomj** [comment]
There are built-in features in the mmdetection framework to use a pre-trained checkpoint, such as  `load_from` or `resume_from`. You may find more info in [the original documentation](https://github.com/open-mmlab/mmdetection/blob/v1.0.0/docs/GETTING_STARTED.md#train-with-multiple-gpus).

**Comment by mrgiba** [comment]
Is it needed to follow the mmdet v1 installation steps at https://github.com/open-mmlab/mmdetection/blob/v1.0.0/docs/INSTALL.md or should mmdet be installed differently ?

The `conda install` step there would install the latest pytorch version, but by the number of errors and deprecation messages I see here when running `pip install -v -e . `, I wonder if the installation steps should restrain pytorch to a specific version instead

**Comment by mrgiba** [comment]
Yes, by restraining to PyTorch 1.4 backwards and also restricting the version of a couple of dependencies, I was able to setup, do transfer learning and run inference. 

I can contribute with the exact installation steps I did
